lib/backpex/html/layout.ex

```diff
@@ -226,6 +220,68 @@ defmodule Backpex.HTML.Layout do
     """
   end
 
+  @doc """
+  Renders a theme selector.
+  """
+  @doc type: :component
+  attr(:socket, :any, required: true)
+
+  attr(:themes, :list,
+    doc: "A list of tuples with {theme_label, theme_name} format",
+    examples: [[{"Light", "light"}, {"Dark", "dark"}]]
+  )
+
+  def theme_selector(assigns) do
+    ~H"""
+    <div id="backpex-theme-selector" class="dropdown dropdown-bottom dropdown-end no-animation">
+      <div tabindex="0" role="button" class="btn m-1">
+        <span class="hidden md:block">
+          <%= Backpex.translate("Theme") %>
+        </span>
+        <Backpex.HTML.CoreComponents.icon name="hero-swatch-solid" class="h-5 w-5 md:hidden" />
```

Comment:
I vote for the outline version of the `swatch` icon because the user icon is also outline.

Reply:
Agree, also using btn-ghost to keep in line with the user icon.